docs/edgeql/sets.rst

```diff
@@ -248,6 +248,43 @@ Use the :eql:op:`union` operator to merge two sets.
   db> select {1, 2} union {3.1, 4.4};
   {1.0, 2.0, 3.1, 4.4}
 
+Finding common members
+----------------------
+
+.. versionadded:: 3.0
+
+Use the :eql:op:`intersect` operator to find common members between two sets.
+
+.. code-block:: edgeql-repl
+
+    db> select {1, 2, 3, 4, 5} intersect {3, 4, 5, 6, 7};
+    {3, 5, 4}
+    db> select {'a', 'b', 'c', 'd', 'e'} intersect {'c', 'd', 'e', 'f', 'g'};
+    {'e', 'd', 'c'}
+
+.. note::
+
+    The ordering of the returned set may not match that of the operands.
+
+
+Finding unique members
+----------------------
+
+.. versionadded:: 3.0
+
+Use the :eql:op:`except` operator to find members in the first set that do not
+appear in the second set.
+
+.. code-block:: edgeql-repl
+
+    db> select {1, 2, 3, 4, 5} except {3, 4, 5, 6, 7};
+    {1, 2}
+    db> select {'a', 'b', 'c', 'd', 'e'} except {'c', 'd', 'e', 'f', 'g'};
+    {'b', 'a'}
+
```

Comment:
Same as for the intersection, it's important to show what happens with multiple copies:
```
db> select {0, 1, 1, 1, 2, 3, 3} except {1, 3, 3, 3, 3, 3};
{0, 1, 1, 2}
```
The number of copies that will be excluded depends on how many copies are present in the second set.